pkg/models/namespace.go

```diff
@@ -80,3 +84,10 @@ const DefaultAnnouncementMessage = `
 *                                                                *
 ******************************************************************
 `
+
+func DefaultEnterprise(msg string) string {
+	if envs.IsEnterprise() {
+		return ""
+	}
+	return msg
+}
```

Comment:
This function will create the namespace with the community message even in cloud instances. You must use either `!envs.IsCommunity()` or `envs.IsEnterprise() && envs.IsCloud()`.